### PR TITLE
chore: lint fixes

### PR DESCRIPTION
**Pull request by caetano-colin**
* Remove trailing whitespaces
* Add newline to EOF
* Ignore some flake8 (python lint) erros
* Fix some flake8 errors
* Remove some pynb checkpoints

### CODE REVIEW

File: 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py
Comment (daniel-cit) [inline 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: why were these files deleted?

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: since this files are autogenerated and serve as a backup for python notebooks, I assumed they would not be useful for users cloning the repository. 

Usually files inside `.ipynb_checkpoints` are untracked.

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: They were also generating a lot of python lint errors, thats the main reason why it stood out

Comment (daniel-cit) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: is `.ipynb_checkpoints`  in the `.gitignore` file ?

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: not in the case of this repo

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: it is an option to keep these files on the repo and ignore all erros on these pynb checkpoint, but I assumed it would be clutter.

Comment (daniel-cit) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: I would add `.ipynb_checkpoints` to the .gitignore file on top of deleting them

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: sure!

Comment (caetano-colin) [reply 7-composer/components/.ipynb_checkpoints/deployment-checkpoint.py:1]: added on commit b54beca6fe409a80c82af753f9d586b851844ae7

**Review by daniel-cit — APPROVED** [review]
LGTM
